src/unit.rs

```diff
@@ -369,7 +369,7 @@ fn send_prelude(unit: &Unit, stream: &mut Stream, redir: bool) -> IoResult<()> {
         }
     }
     if !header::has_header(&unit.headers, "user-agent") {
-        write!(prelude, "User-Agent: ureq\r\n")?;
+        write!(prelude, "User-Agent: rust-ureq/{}\r\n", env!("CARGO_PKG_VERSION"))?;
```

Comment:
```suggestion
        write!(prelude, "User-Agent: ureq/{}\r\n", env!("CARGO_PKG_VERSION"))?;
```

Reply:
I'm in awe of your detail knowledge :)

Let's drop the `rust` bit completely.

https://github.com/algesten/hreq/blob/master/src/lib.rs#L388

Reply:
Thank you! It's not so much detailed knowledge as "that looks funny; I wonder why I've never seen a UA like that?" And then going and Ctrl-F'ing in the spec. :-)